openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -882,34 +882,35 @@ Cache-Control: no-store
 
 When the Credential Request is invalid or unauthorized, the Credential Issuer constructs the error response as defined in this section.
 
-The following additional clarifications are provided for the following parameters already defined in section 3.1 of [@!RFC6750]:
+#### Authorization Errors {#authorization-errors}
 
-`invalid_request`:
+If the Credential Request does not contain an Access Token that enables issuance of a requested credential, the Credential Endpoint returns an authorization error response such as defined in section 3 of [@!RFC6750].
```

Comment:
```suggestion
If the Credential Request does not contain an Access Token that enables the issuance of a requested credential, the Credential Endpoint MUST use the HTTP status code 401 (Unauthorized) with the content type set to `application/json` and the error code set to `invalid_token`, as defined in section 3 of [@!RFC6750].
```

Reply:
`invalid_token` error code should not be mandated. and it is not application/json. and 401 is already covered by `as defined by rfc6750`.

```suggestion
If the Credential Request does not contain an Access Token that enables issuance of a requested credential, the Credential Endpoint MUST return an authorization error response as defined in section 3 of [@!RFC6750] .
```

Reply:
> invalid_token error code should not be mandated. and it is not application/json. and 401 is already covered by as defined by rfc6750.

+1 

but changing it to have that MUST would preclude challenging for dpop bound access tokens. The text in question here shouldn't change.

Reply:
please put this back to "If the Credential Request does not contain an Access Token that enables issuance of a requested credential, the Credential Endpoint returns an authorization error response such as defined in section 3 of [@!RFC6750]."